docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst

```diff
@@ -242,6 +242,8 @@ Name                                                Description
 ``pool.scheduled_tasks``                            Number of scheduled tasks in the pool. Metric with pool_name tagging.
 ``pool.starving_tasks.<pool_name>``                 Number of starving tasks in the pool
 ``pool.starving_tasks``                             Number of starving tasks in the pool. Metric with pool_name tagging.
+``task.cpu_usage_percent.<dag_id>.<task_id>``       Percentage of CPU used by a task
```

Comment:
I don't think we need "_percent". 
also, given this notation, does it mean we emit multiple DAG runs aggregated into 1 metric? What happens when multiple DAG runs are active? I am not against skipping <dag_run_id> as it will save cost, but curious if you thought about this case?

Reply:
> I don't think we need "_percent".

100 % fine

> also, given this notation, does it mean we emit multiple DAG runs aggregated into 1 metric? What happens when multiple DAG runs are active? I am not against skipping <dag_run_id> as it will save cost, but curious if you thought about this case?

The task ID is enough, the task ID is unique already. The `dag_id` is here only so that it simpler to find out in which DAG the task is running. We could drop `dag_id` is you think this is not necessary. I also followed what is being done today, every metric on task level includes the `dag_id`.

Reply:
Interesting, I didn't realize that task_id is unique. It's good to keep dag_id as it gives easy way to filter by DAG name.

Reply:
`task_id` is unique only within the DAG.

Reply:
Ah my bad! That explains why we need the dag ID then